apiserver/storage/storage.go

```
@@ -14,55 +14,61 @@ import (
 	"github.com/juju/juju/apiserver/common/storagecommon"
 	"github.com/juju/juju/apiserver/facade"
 	"github.com/juju/juju/apiserver/params"
+	"github.com/juju/juju/environs"
 	"github.com/juju/juju/state"
+	"github.com/juju/juju/state/stateenvirons"
 	"github.com/juju/juju/status"
 	"github.com/juju/juju/storage"
 	"github.com/juju/juju/storage/poolmanager"
-	"github.com/juju/juju/storage/provider/registry"
 )
 
 func init() {
-	common.RegisterStandardFacade("Storage", 2, NewAPI)
+	common.RegisterStandardFacade("Storage", 2, newAPI)
```

Perhaps an api bump so older beta clients fail explicitly